kit/popups/src/UIInteractiveNotice/CloseButton.tsx

```diff
@@ -0,0 +1,40 @@
+import * as React from 'react';
+import { ViewStyle, View } from 'react-native';
+import { UIAssets } from '@tonlabs/uikit.assets';
+import { UIPressableArea } from '@tonlabs/uikit.controls';
+import { UILayoutConstant } from '@tonlabs/uikit.layout';
+import { UIImage } from '@tonlabs/uikit.media';
+import { ColorVariants } from '@tonlabs/uikit.themes';
+import type { InteractiveNoticeContentProps } from './types';
+
+export function CloseButton({
+    style,
+    showCloseButton,
+    onClose,
+}: Pick<InteractiveNoticeContentProps, 'showCloseButton' | 'onClose'> & { style: ViewStyle }) {
+    if (!showCloseButton) {
+        return null;
+    }
+    return (
+        <View style={style}>
+            <UIPressableArea
+                onPress={onClose}
+                style={{
+                    marginVertical: -UILayoutConstant.contentInsetVerticalX4,
+                    marginHorizontal: -UILayoutConstant.contentOffset,
```

Comment:
Same question. Besides, does it work well with Android where all the Views have overflow: 'hidden'?

Reply:
It works well on all platforms, I checked.